src/test/java/com/epam/healenium/SelenideTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Healenium-web Copyright (C) 2019 EPAM
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.healenium;
+
+import static com.codeborne.selenide.Condition.value;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.open;
+import static org.openqa.selenium.firefox.FirefoxDriver.PROFILE;
+
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.WebDriverProvider;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class SelenideTest {
+
+    private static final int PORT = 8090;
+
+    @Rule
+    public TestServer server = new TestServer("SomeNewRoot", PORT);
+
+    @Before
+    public void setUp() {
+        System.setProperty("webdriver.gecko.driver", "C:\\projects\\healing\\drivers\\geckodriver.exe");

```

Comment:
will not pass in CI due to hardcoded path.     
WebDriverManager.firefoxdriver().setup(); is better